docs/interfaces/webui-if.rst

```diff
@@ -71,6 +76,43 @@ visit :ref:`users`.
 With the Determined Enterprise Edition, you can also create and manage user groups. To learn more,
 visit :ref:`rbac`.
 
+************************
+ Managing User Settings
```

Comment:
Not sure this whole section is really meaningful? You're just kind of listing the sections in the settings drawer. I'd expect sections based on actions the user wants to take rather than essentially just duplicating the settings drawer as prose. The only actual documentation here seems to be the first two paragraphs that tell you where to find the settings.

Reply:
@ClaireNeveu thanks for the feedback. i've made some changes to keep it concise.